sydent/server.py

```diff
@@ -0,0 +1,81 @@
+from zope.interface import implementer
```

Comment:
Shouldn't this file be under `tests`? It looks test-related to me.

Reply:
It is test related, but @erikjohnston seemed to think it belonged here instead. Possibly because of Synapse reasons?